### ISSUE TITLE
[$15][MSFT-209] Sample challenge page: The 'arrow' which when clicked takes the user back to the challenge feed page is being read as 'ico-arrow-big-left' by screen readers.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2872]
**Steps to Reproduce**
1. Go to the sample challenge page: https://www.topcoder.com/challenges/30095347. 2. Ensure the screen reader is switched on. 3. Press the tab key from the 'find members by username or skill' textbox. 4. Note the screen reader reads 'link graphic ico-arrow-big-left'. 

**Expected Result**
The label for this link should be 'back' indicating its purpose.

**Actual Result**
The label being read by the screen reader for the back link is inappropriate.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.4 Link Purpose (In Context)
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2036

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095981 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095981 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2872 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2872 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2872 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2872 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2872 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2872 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2872 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2872 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2872 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2872 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors this is open for pickup.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097891 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097891 has been updated - it has been assigned to mfikria.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by tc-mfikria** [comment]
@veshu resolved in https://github.com/topcoder-platform/community-app/pull/3148

**Comment by r0hit-gupta** [comment]
@tc-mfikria the tests are failing. Please update the PR. Thanks

**Comment by sushilshinde** [comment]
@topcoder-platform/topcodercompetitors this is open for pickup.

PR on `qa-accessibility` branch

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097891 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by rashmi73** [comment]
@veshu PR https://github.com/topcoder-platform/community-app/pull/3179

**Comment by drasticdpk** [comment]
Verified on Windows/Chrome/Jaws

**Comment by lakshmiathreya** [comment]
@drasticdpk @rashmi73 - pls clarify - The arrow is read out as 'main landmark ico-arrow-big-left' - tested on Prod Win10/Chrome/NVDA. On MacOS/Chrome/Voice Over it is ready out as 'ico-arrow-big-left main' @tosha5252 @sushilshinde fyi.

**Comment by rashmi73** [comment]
@lakshmiathreya it was working on windows chrome well, can you please check if it is a deployment issue again?

**Comment by crazyk07** [comment]
@nkumar-topcoder  Just checked as well, still no change.  

@Oanh-and-only-Oanh  What do you think should be the exact wording for the back button should be?

**Comment by Oanh-and-only-Oanh** [comment]
Should read as "Arrow icon back to Challenge Listing page"

**Comment by veshu** [comment]
@sushilshinde This code is not deployed to production. please check. Thanks!

**Comment by tosha5252** [comment]
reads 'ico-arrow-link-left-link'

![arrow](https://user-images.githubusercontent.com/6400904/62685164-bcc10280-b987-11e9-8846-43a529cdccfc.png)

Tested using JAWS
Windows 10 env
Browser: Chrome

**Comment by crazyk07** [comment]
@tosha5252 Try again, think you need to use incognito

**Comment by tosha5252** [comment]
Checked in incognito mode...readout is appropriate..."back to challenge list link"

Win10/Chrome/JAWS

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097891<br/><br/>```This is an automated message for crazyk via Topcoder X```
